app/jobs/daily_regeneration_job.rb

```diff
@@ -0,0 +1,31 @@
+class DailyRegenerationJob < ApplicationJob
+  queue_as :regeneration
+
+  def perform(school:)
+    GoodJob.logger.info("#{DateTime.now.utc} Regeneration for school #{school.name} start")
+    run_validate_and_persist_readings_service_for(school)
+    run_aggregate_school_service_for(school)
+    run_content_batch_for(school)
+    GoodJob.logger.info("#{DateTime.now.utc} Regeneration for school #{school.name} end")
+  end
+
+  def run_aggregate_school_service_for(school)
```

Comment:
maybe rename this and the log message, its not aggregating the school here, just invalidating the rails cache. I've never seen it thrown an error, but no harm in having the exception handler